onboarding-enabler-spring-sample-app/build.gradle

```diff
@@ -36,16 +33,16 @@ repositories {
 }
 
 dependencies {
-    compile(project(':onboarding-enabler-spring'))
-    compile "org.springframework.boot:spring-boot-starter-actuator:${springBootVersion}"
-    compile "io.springfox:springfox-swagger2:2.9.2"
-    compile "io.springfox:springfox-spring-web:2.9.2"
+    implementation(project(':onboarding-enabler-spring'))
+    implementation libraries.spring_boot_starter_actuator
+    implementation "io.springfox:springfox-swagger2:2.9.2"
+    implementation "io.springfox:springfox-spring-web:2.9.2"
```

Comment:
what is the reason to update springfox if we want to get rid of it?

Reply:
#2427 will use versions.gradle for springdoc, and remove the springfox references